shell/platform/embedder/embedder.h

```diff
@@ -1192,6 +1192,17 @@ FlutterEngineResult FlutterEngineOnVsync(FLUTTER_API_SYMBOL(FlutterEngine)
                                          uint64_t frame_start_time_nanos,
                                          uint64_t frame_target_time_nanos);
 
+//------------------------------------------------------------------------------
+/// @brief      Reloads the system fonts in engine.
+///
+/// @param[in]  engine.                  A running engine instance.
+///
+/// @return     The result of the call.
+///
+FLUTTER_EXPORT
+FlutterEngineResult FlutterEngineReloadSystemFonts(
```

Comment:
What happens if someone calls this but doesn't send the platform event? It seems like if both are required that should be documented.

Although it would be even better if that were handled internally. Couldn't the implementation send a system channel message? `shell/common/shell.cc` uses the `flutter/skia` channel for instance.

Reply:
I can't think of a use case at the moment where someone only reload the system fonts but don't want to rebuild flutter app .
I should be able to combine them